src/test/java/LoginPageTest.java

```diff
@@ -60,6 +60,60 @@ public void loginPage_LoginFormSection_HasIntroductionAndLoginFormCorrectlyDispl
         softAssertions.assertAll();
     }
 
+    @Test
+    public void loginPage_HeaderFooterNavigationDemoCustomer_AreSectionsCorrectlyDisplayed() {
+        //given
+        LoginPage loginPage = new LoginPage(driver);
+
+        //when
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(loginPage.isPageHeaderDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isPageNavigationBarDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isPageFooterDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isNewCustomerContainerDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isTryDemoCustomerAccessSectionDisplayed()).isTrue();
```

Comment:
If there are test for new customer section and try demo customer access section, then it would be the best to move validations of the presence of section move to these tests.